docs/UserGuide.md

```diff
@@ -464,14 +477,19 @@ Format: `exit`
 It is just a visual difference. Both applications have the same functionalities otherwise.
 
 <div style="page-break-after: always;"></div>
+
 **Q**: What are tags used for? <br>
 **A**: Tags can be used to indicate important details of the tutee that you would like to see straight from the `tuteelist`. For instance, in the sample data provided, tags were used to indicate important upcoming examinations for tutees such as `PSLE`. 
 
+**Q**: What does it mean by a lesson clashing with other lessons under the section on [adding a lesson to a tutee](#adding-a-lesson-to-a-tutee--addlesson)? Why is it not allowed?<br>
+**A**: This means that the lesson is occurring at the same day and the same time as other lessons. This is not allowed as it is not feasible for a tutor to have two or more ongoing lessons at the same time.
```

Comment:
```suggestion
**A**: A schedule clash means that the lesson added coincides with another lesson in the existing schedule. To resolve this, pick a different day/timing where your schedule is free.
```

Reply:
I think can just focus on how to resolve versus explaining to them why its not feasible, since its straightforward